Create Gitpod vs JetBrains landing page

nisarhassan12 opened the pull request:
Fixes #1363 

Deploy preview: https://deploy-preview-1366--gitpod-kumquat.netlify.app/vs/jetbrains-space

This is how it looks:

<details>
<summary>
Screenshot
</summary>
<img src="https://user-images.githubusercontent.com/46004116/146609479-c529c08c-72dd-46b9-bf5d-6bc20d442b17.png" />
</details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1366"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

ChristinFrohne reviewed (commented):
Thanks, here is some more feedback:
- [x] footer: JetBrains is spelled with a capital B 
- [ ] footer & URL: It should be JetBrains Space

- [x] try to avoid single words i a new paragraph e.g. "your craft"
- [x] the JetBrains link underneath in the footer of the pricing graphic should link to JetBrains Space Pricing (see mockup)
- [ ] I created a mockup for mobile view, please use that: Adjust the screenshot, align the text in the feature cards to the left and move the image before the text (we already discussed that on the mockup)

and I see that the feature table is still missing. Once you finished the implementation and addressed the feedback, please ping Johannes for a final review before merging it.

nisarhassan12 commented on src/routes/vs/jetbrains-space.svelte:
```diff
@@ -0,0 +1,40 @@
+<OpenGraph
+  data={{
+    description:
+      "Gitpod is the vendor neutral, open-source platform that integrates with your individual stack including all JetBrains IDEs.",
```
For the description, I just took the text above the CTA in the hero area. Let me know in case we want to use something else.

nisarhassan12 commented:
> Once you finished the implementation and addressed the feedback, please ping Johannes for a final review before merging it.
Hi @JohannesLandgraf please do the final review when you get time. Thanks

mikenikles commented:
I think @loujaybee can help out as well.

phimae commented:
Looking at the preview again, I was wondering if the cost efficiency screen should be the first box as it might get the reader hooked since it's such a strong difference? cc @JohannesLandgraf since you're tagged for final review

![image](https://user-images.githubusercontent.com/83109781/146753884-7b14c299-02dd-4ac8-bd5c-2410526eb51c.png)

nisarhassan12 commented:
I'm merging this now because we are gonna need to have 0 open PRs by the by end of day Wednesday, December 22 as stated here https://gitpod.slack.com/archives/C01KGM9ETU6/p1639788614237000

The reviewers can add review comments here and once those review comments are resolved we need to revert https://github.com/gitpod-io/website/pull/1366/commits/b0f12c1f2e79987fe1816126ad36a7db810e3085 in a follow up Pr.

What https://github.com/gitpod-io/website/pull/1366/commits/b0f12c1f2e79987fe1816126ad36a7db810e3085 does is that it hides the link to the JetBrains page and also makes sure that it is not accessible by search engines.
